Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -545,6 +545,16 @@ public void BeginInvokeOnMainThread(Action action)
 				s_handler.Post(action);
 			}
 
+			public void BeginInvokeOnMainThread(Action action, Guid windowId)
```

Comment:
@wachs Are these necessary anymore? Or can we clean them up?

Reply:
Actually the codes are old and no longer needed.